app/src/main/java/com/gdfp/android_evaluationv2/MainActivity.java

```diff
@@ -89,10 +92,10 @@ public void onClick(View v) {
     public void onItemClick(View v, int position) {
         //TODO: Handle list item click event
         // Create an Intent to navigate to the TaskDetailActivity
-
+        Intent intent = new Intent(this,TaskDetailActivity.class);
 
         //TODO: Set the data (URI and item Id) in the Intent
-
+        //intent.putExtra();
```

Comment:
The idea for this is instruction is to use ``` detailItent.setData()``` please take a look to this method with detailIntent.